app/apollo/resolvers/subscription.js

```diff
@@ -672,16 +680,19 @@ const subscriptionResolvers = {
         */
 
         const conditions = await getGroupConditionsIncludingEmpty(me, org_id, ACTIONS.READ, 'name', queryName, context);
-        logger.debug({req_id, user, org_id, conditions }, `${queryName} group conditions are...`);
-        var subscription = await models.Subscription.findOne({ org_id, uuid, ...conditions }, {}).lean({ virtuals: true });
+        logger.debug({req_id, user, org_id, conditions}, `${queryName} group conditions are...`);
+
+        let subscription = await models.Subscription.findOne({ org_id, uuid, ...conditions }, {}).lean({ virtuals: true });
+        logger.info({req_id, user, org_id, uuid, version_uuid}, `${queryName} validating - found: ${!!subscription}`);
 
-        if(!subscription){
+        const identifiers = subscription ? [uuid, subscription.name] : [uuid];
+        await validAuth(me, org_id, ACTIONS.SETVERSION, TYPES.SUBSCRIPTION, queryName, context, identifiers);
+        logger.info( {req_id, user, org_id, uuid, version_uuid}, `${queryName} validating - authorized` );
+
```

Comment:
Should READ auth to the config (channel) also be verified?

Reply:
fixed